Documentation/operation docs rework (#3074)
scality/metalk8s: docs/operation/downgrade.rst

> MetalK8s Cluster Downgrade

Reviewer: If you agree with the previous comment on section titles:

```suggestion
Cluster Downgrade
```

==========================

> The MetalK8s Cluster Downgrade is handled via utility scripts which are

Reviewer 2: ```suggestion
The MetalK8s cluster downgrade is handled using utility scripts that are
```

      Version |release| only supports downgrade of patch version.

Prerequisites
*************

> ISO Preparation

Reviewer 3: only this step under "Prerequisites"? 
Remove this title then

---------------

Provision a new **MetalK8s** ISO by running the utility script shipped
with the current installation.

.. code::

   /srv/scality/metalk8s-X.X.X/iso-manager.sh -a <path_to_iso>

Pre-checks
----------

You can test if your environment will successfully downgrade with the following
command.

.. code::

   /srv/scality/metalk8s-X.Y.Z/downgrade.sh --destination-version \
     <destination_version> --dry-run --verbose

This will simulate the downgrade pre-checks and provide an overview of the
changes to be carried out in your MetalK8s cluster.

.. important::

>     The version prefix metalk8s-**X.Y.Z** must be the *current* installed

Reviewer 2: this should be handled with a variable, no?

Reviewer: Relates to a previous comment: I think yes. Would make the procedures much simpler to follow, assuming we clearly express their scope in a preamble and let users know where to find the correct one if it doesn't match their goal (e.g. downgrade "to" the documented version would need to run another procedure, from the documentation of the currently installed version).

Maybe worth reworking all of this in another PR, your call @cathydossantospinto :wink:

Reviewer: Opened #3184

    MetalK8s version.

Downgrade
*********

> #. Ensure that the prerequisites have been met.

Reviewer 3: Remove

#. Run the utility script shipped with the *current* installation
   providing it with the destination version.

#. From the :term:`Bootstrap node`, launch the downgrade.

>    .. code::

Reviewer 2: parsed-literal and variable?

      /srv/scality/metalk8s-X.Y.Z/downgrade.sh --destination-version <version>

> .. important::

Reviewer 2: If you can insert the version variable, you can eliminate this message.
